parsers/fortran.c

```diff
@@ -582,10 +582,11 @@ static void makeLabelTag (vString *const label)
 
 static lineType getLineType (void)
 {
-	vString *label = vStringNew ();
+	static vString *label = NULL;
```

Comment:
@masatake yes I meant what Valgrind reports as "still reachable".  And no, depending on the context I don't really see it as a leak, hence the quotes around the word.

So yes, we can totally decide it's fine, but it can be a slight imperfection if/when ctags parsers become a library, because it can then long-live, and a caller could want to limit resource usage and deinit parsers as soon as it can reasonably think it doesn't need them anymore soon (maybe when closing the last file of that filetype in an editor).

Anyway, if we have this pattern in many places already, it doesn't matter in the context of this PR, so if it should be a discussion at all, it probably should happen on its own.